jtransc-rt-core/src/com/jtransc/JTranscSystem.java

```diff
@@ -104,7 +104,7 @@ static public void debugger() {
 	@HaxeMethodBody("if (!p0) N.debugger();")
 	@JTranscMethodBodyList({
 		@JTranscMethodBody(target = "js", value = "if (!p0) debugger;"),
-		@JTranscMethodBody(target = "cs", value = "if (!pc) System.Diagnostics.Debugger.Break();"),
+		@JTranscMethodBody(target = "cs", value = "if (!p0) System.Diagnostics.Debugger.Break();"),
```

Comment:
Ok. No problem. Then a test could be just like:

```
class BoolLiteralTest {
    static private final boolean TRUE = true;
    static private final boolean FALSE = false;

    static public void main(String[] args) {
         System.out.println(TRUE);
         System.out.println(FALSE);
    }
}
```

And you can call it from BigTest class.

https://github.com/jtransc/jtransc/blob/e880096b22490e22487817bf3380528bdd5c5b92/jtransc-main/test/big/BigTest.kt#L38

Then running BigTests in C# should make it fail, right?